XRTK-Core/Packages/com.xrtk.core/Tests/Core/TestFixture_01_MixedRealityToolkitTests.cs

```diff
@@ -617,6 +624,43 @@ public void Test_05_09_GetAllMixedRealityExtensionServices()
             Assert.IsTrue(allExtensionServices.Count == 4);
         }
 
+        [Test]
+        public void Test_05_10_RegisterMixedRealityExtensionServiceAndDataProvider()
+        {
+            TestUtilities.InitializeMixedRealityToolkitScene(false);
+
+            var activeSystemCount = MixedRealityToolkit.ActiveSystems.Count;
+            var activeServiceCount = MixedRealityToolkit.RegisteredMixedRealityServices.Count;
+
+            // Register
+            MixedRealityToolkit.TryRegisterService<ITestExtensionService1>(new TestExtensionService1("Test17-1.1"));
```

Comment:
Could we use a more meaningful name for the service?

Reply:
It is using the naming conventions set down by the initial person who wrote the tests.  As it is just a test, the naming is not that critical, other than to state it is an "Extension" service.

If you wish to change all the names of all the definitions, please feel free to submit a PR

Reply:
I just think 17-1.1 doesn't really make sense. The test isn't even the 17th test. Just curious where 17-1.1 came from is all and how it relates to the name of the test service instance.

Reply:
imho since it's test 5.10 it almost makes more sense if the name was `Test5.10.1` or something

Reply:
The numbering simply followed on from the numbering you implemented in the previous test.